tests/unit/backend/utils/field-validation/attachment-validation.spec.ts

```diff
@@ -0,0 +1,174 @@
+import { ValidateFieldError } from 'src/app/modules/submission/submission.errors'
+import { ProcessedAttachmentResponse } from 'src/app/modules/submission/submission.types'
+import { validateField } from 'src/app/utils/field-validation/'
+import { IFieldSchema } from 'src/types'
+
+type MakeFieldOptions = {
+  required?: boolean
+}
+
+type MakeResponseOptions = {
+  answer?: string
+  filename?: string
+}
+
+describe('Attachment validation', () => {
+  const makeField = (
```

Comment:
possible to use `generateDefaultField` from `helpers/generate-form-data.ts`?

Reply:
more generally, I prefer defining helper functions outside the test blocks to keep the tests more readable